external/anomaly/configs/base/configuration.py

```diff
@@ -19,10 +19,7 @@
 from sys import maxsize
 
 from attr import attrs
-from ote_anomalib.configs.configuration_enums import (
-    ModelBackbone,
-    POTQuantizationPreset,
-)
+from configs.utils.configuration_enums import ModelBackbone, POTQuantizationPreset
```

Comment:
I think this should be changed to
`from configs.base.configuration_enums import ...`